module-apps/application-phonebook/data/PhonebookItemData.hpp

```diff
@@ -61,13 +61,26 @@ class PhonebookSearchRequest : public gui::SwitchData
   public:
     std::string request                                 = "";
     std::shared_ptr<std::vector<ContactRecord>> results = nullptr;
+    unsigned int selectedNumber                         = 1;
```

Comment:
why in the newMessage window the `selectedNumber` defaults to 0, and here to 1?

Reply:
ups -> my mistake of course.